Sming/Makefile-rboot.mk

```diff
@@ -12,6 +12,9 @@
 RBOOT_BIG_FLASH  ?= 1
 RBOOT_TWO_ROMS   ?= 0
 RBOOT_RTC_ENABLED ?= 0
+RBOOT_GPIO_ENABLED ?= 0
+RBOOT_IROM_CHKSUM ?= 0
```

Comment:
enabling this requires an extra option on the call to esptool2 that creates the rom image (to include the data in the checksum)